vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -207,7 +207,14 @@ components:
 
     CpuConfig:
       required:
-      - cpu_count
+      - boot_vcpus
+      type: object
+      properties:
+        cpu_count:
```

Comment:
I think that's a mistake, and it should be `boot_vcpus` instead.
And same thing a few lines below where it should be `max_vcpus` instead of `cpu_count`.